### PR TITLE
docs for ssl update change

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

File: _deployment/ssl.md [hunk 1]
```diff
@@ -28,23 +28,42 @@ Your app is now configured to serve encrypted traffic with a self-signed certifi
 
 ### Upload your certificate
 
-Use the Convox CLI to upload your certificate and private key, specifying the process and external port you want the certificate applied to. Continuing with the previous example, the command would look like:
+Upload you certificate using `convox certs create`:
```
Comment (mwarkentin) [inline _deployment/ssl.md]: `you` -> `your`

**Comment by mwarkentin** [comment]
Maybe worth a small section about how to use this with ACM certs?

**Comment by ddollar** [comment]
Added a section on SSL cert generation and fixed the typo. Thanks!

**Comment by mwarkentin** [comment]
Maybe add a note that it's only available in certain regions and link to http://docs.aws.amazon.com/acm/latest/userguide/acm-regions.html?

**Comment by ddollar** [comment]
Great link. Added and thanks :)

**Pull request merged**